src/sql/platform/connection/common/connectionProfile.ts

```diff
@@ -3,17 +3,15 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-'use strict';
```

Comment:
We can get rid of all of these since you enabled strict checks a little while back, right?

Reply:
We have been able to remove these for a while, these don't do anything since we have https://github.com/Microsoft/azuredatastudio/blob/master/src/tsconfig.base.json#L11